mars/services/task/execution/ray/executor.py

```diff
@@ -154,11 +173,19 @@ def __init__(
 
         self._available_band_resources = None
 
-        # For progress
+        # For progress and task cancel
         self._pre_all_stages_progress = 0.0
-        self._pre_all_stages_tile_progress = 0
-        self._cur_stage_tile_progress = 0
+        self._pre_all_stages_tile_progress = 0.0
+        self._cur_stage_progress = 0.0
+        self._cur_stage_tile_progress = 0.0
+        self._cur_stage_subtask_to_output_object_refs = dict()
         self._cur_stage_output_object_refs = []
```

Comment:
It seems that the `_cur_stage_output_object_refs` is unused any more.

Reply:
Thanks for your reminder. I'll remove it.